### PR TITLE
Add rbd dynamic provisioning section

### PR DESCRIPTION
**Pull request by rootfs**
@leseb @font

### CODE REVIEW

File: examples/kubernetes/create_secrets.sh [hunk 1]
```diff
@@ -10,5 +10,6 @@ kubectl create secret generic ceph-bootstrap-rgw-keyring --from-file=ceph.keyrin
 kubectl create secret generic ceph-bootstrap-mds-keyring --from-file=ceph.keyring=ceph.mds.keyring --namespace=ceph
 kubectl create secret generic ceph-bootstrap-osd-keyring --from-file=ceph.keyring=ceph.osd.keyring --namespace=ceph
 kubectl create secret generic ceph-client-key --from-file=ceph-client-key --namespace=ceph
+kubectl create secret generic ceph-secret-admin --from-file=generator/ceph-client-key --type=kubernetes.io/rbd --namespace=ceph
```
Comment (font) [inline examples/kubernetes/create_secrets.sh]: No need to specify the `generator` directory here as we `cd generator` above.

Comment (rootfs) [inline examples/kubernetes/create_secrets.sh]: nice catch!
fixed

**Comment by rootfs** [comment]
ping @leseb, PTAL thanks

**Comment by guits** [comment]
jenkins test ceph_ansible-jewel-centos7-cluster

**Comment by guits** [comment]
jenkins test ceph_ansible-luminous-centos7-cluster

**Comment by guits** [comment]
jenkins test ceph_ansible-luminous-centos7-cluster

**Pull request merged**